Makefile

```diff
@@ -123,7 +123,7 @@ configure: configure.ac aclocal.m4 build-aux/ocaml_version.m4 tools/autogen
 
 .PHONY: partialclean
 partialclean::
-	rm -f utils/config.ml utils/domainstate.ml utils/domainstate.mli
+	$(MAKE) -C utils partialclean
```

Comment:
This is the only one I disagree with - `utils/Makefile` only exists for the Dune build - the `utils/` is not an independent unit in the way all the other directory `Makefile`s are.